apps/dashr-opiod-endemic/app.R

```diff
@@ -0,0 +1,383 @@
+library(dashR)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+library(purrr)
+library(dplyr)
+library(stringr)
+library(glue)
+library(data.table)
+library(plotly)
+library(readr)
+
+appName <- Sys.getenv("DASH_APP_NAME")
+pathPrefix <- sprintf("/%s/", appName)
+
+Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix,
+           DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)
+
+setwd("/app/apps/dashr-opiod-endemic")
+
+app <- Dash$new()
+
+# get lat long data and transform
+df_lat_lon <- as.data.frame(fread("lat_lon_counties.csv"))
+df_lat_lon$FIPS <- str_pad(df_lat_lon$FIPS, 5, pad = "0")
+
+# get lat long data and transform
+df_full_data <- as.data.frame(fread("age_adjusted_death_rate_no_quotes.csv"))
+df_full_data$`County Code` <- str_pad(df_full_data$`County Code`, 5, pad = "0")
+df_full_data$County <- paste0(df_full_data$V2, ", ", df_full_data$County)
+df_full_data <- df_full_data[, -c(1,2)]
+df_full_data$`Age Adjusted Rate` <- replace(df_full_data$`Age Adjusted Rate`, 
+                                            df_full_data$`Age Adjusted Rate` == "Missing", "")
+df_full_data$Deaths <- replace(df_full_data$Deaths, df_full_data$Deaths == "Missing", "")
+df_full_data$`Age Adjusted Rate` <- sub("(Unreliable)", "", df_full_data$`Age Adjusted Rate`, 
+                                       ignore.case = TRUE)
+df_full_data$`Age Adjusted Rate` <- parse_number(as.character(df_full_data$`Age Adjusted Rate`))
+df_full_data$Deaths <- parse_number(as.character(df_full_data$Deaths))
+df_full_data$Deaths <- replace(df_full_data$Deaths , is.na(df_full_data$Deaths), 0)
+df_full_data$`Age Adjusted Rate` <- replace(df_full_data$`Age Adjusted Rate`, 
+                                            is.na(df_full_data$`Age Adjusted Rate`), 0)
+
+YEARS = c(2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2015)
```

Comment:
`YEARS <- 2003:2015`

Also best to stay consistent with using `<-` for assignment instead of `=`